devops

```diff
@@ -0,0 +1 @@
+#placeholder
```

Comment:
What is the purpose of this top level file?

Reply:
Devops has had architecture related topics in the past.  Example:  Nexus organization